frontend/app/templates/components/background-component/def-image-component.hbs

```diff
@@ -1,18 +1,33 @@
 <div class="field">
     <label>Select from default backgrounds</label>
     {{#ui-dropdown class="selection" selected="Bob" onChange=(action (mut image))}}
-        <div class="default text">Select background from list</div>
-        <i class="dropdown icon"></i>
-        <div class="menu">
-            {{#each defImages as |image|}}
+    <div class="default text">Select background from list</div>
+    <i class="dropdown icon"></i>
+    <div class="menu">
+        <br>
+        <div style="margin-left: 10px;" class="ui toggle checkbox listview">
+                <input type="checkbox" name="public" onchange={{action 'change'}}>
+            <label>List view</label>
+        </div>
+        <br><br> {{#if listview}} {{#each defImages as |image|}}
```

Comment:
If you could split it into two lines, they would look great.

Reply:
I was using a document formatter but anyways have updated the indentation. :)